docs/src/gs/create-workload-cluster.md

```diff
@@ -52,16 +58,20 @@ clusterctl generate cluster <cluster-name>\
 
 ## Create a new workload cluster on bare metal instances using an Ubuntu custom image
 
-Note the addition of `OCI_PV_TRANSIT_ENCRYPTION=false` which is required for most BM shapes.
+Note the addition of the `OCI_CONTROL_PLANE_SHAPE` variables, `OCI_WORKER_SHAPE` variables to change the shape 
```

Comment:
This needs to be reworded to something like:

You can specify different shapes for the control plane and worker nodes separately using the `OCI_CONTROL_PLANE_SHAPE` and  `OCI_WORKER_SHAPE` variables.